pkgs/top-level/python-packages.nix

```diff
@@ -13199,6 +13199,17 @@ self: super: with self; {
 
   quaternion = callPackage ../development/python-modules/quaternion { };
 
+  qubes-core-admin = callPackage ../development/python-modules/qubes-core-admin { };
+
+  qubes-core-admin-client = callPackage ../development/python-modules/qubes-core-admin-client { };
+
+  qubes-core-qrexec = callPackage ../development/python-modules/qubes-core-qrexec { };
+
+  qubes-core-qubesdb = toPythonModule (pkgs.qubes-core-qubesdb.override {
+    inherit python;
+    withPython = true;
```

Comment:
Might be better to make the main derivation have `withPython ? true,` and drop the override.

Reply:
Well, we still need to build this package using specific python version, thus override is inevitable